src/opyv8/model.py

```diff
@@ -54,6 +54,7 @@ class Label:
     width: int
     height: int
     classifier: str
+    confidence: int
```

Comment:
This should be of type float.

Reply:
yea.........

Reply:
fixed it

Reply:
thanks for pointing it out

Reply:
No problem, I will take a further look at it and release if everything works as expected. Again, thanks for your PR.